### PR TITLE
Screenstats

### PR DESCRIPTION
**Pull request by mauricewillemsen**
What done:
Removed all combo variables and methods from Score
Added amount of bubbles dropped and popped to Score
Added those variables to the playerobserver.
Added those variables to ScoreScreen and GameData
Changed the tests.
Added timekeeper to Game
Display timekeeper
Fixed issue #134

### CODE REVIEW

File: bust-a-move-framework/src/main/java/bustamove/game/GameData.java [hunk 1]
```diff
@@ -142,6 +144,10 @@ public final void render(final GameContainer container,
                 GameConfig.MARGIN_LEFT + this.offset, GameConfig.SECOND_LINE);
         g.drawString("Name:" + this.namePlayer, GameConfig.MARGIN_LEFT
                 + this.offset, GameConfig.FIRST_LINE);
+        g.drawString("Dropped:" + this.droppedBubbles, GameConfig.MARGIN_LEFT
```
Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/bustamove/game/GameData.java]: I though we only show this stat in the ScoreScreens.

File: bust-a-move-framework/src/main/java/bustamove/game/GameData.java [hunk 2]
```diff
@@ -142,6 +144,10 @@ public final void render(final GameContainer container,
                 GameConfig.MARGIN_LEFT + this.offset, GameConfig.SECOND_LINE);
         g.drawString("Name:" + this.namePlayer, GameConfig.MARGIN_LEFT
                 + this.offset, GameConfig.FIRST_LINE);
+        g.drawString("Dropped:" + this.droppedBubbles, GameConfig.MARGIN_LEFT
+                + this.offset, GameConfig.THIRD_LINE);
+        g.drawString("Popped:" + this.poppedBubbles, GameConfig.MARGIN_LEFT
```
Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/bustamove/game/GameData.java]: *

File: bust-a-move-framework/src/test/java/bustamove/player/PlayerTest.java [hunk 3]
```diff
@@ -50,16 +48,9 @@ public void setName() {
...
-        player.setCombo(2);
-        assertThat("Combo should be set", player.getCombo(), equalTo(2));
-    }
-
-    @Test
     public void reset() {
         player.reset();
-        assertThat("Combo should be 1", player.getCombo(), equalTo(1));
-        assertThat("Score should be 0", player.getScore().getScore(), equalTo(0));
+        assertThat("Dropped should be 0", player.getScore().getScore(), equalTo(0));
```
Comment (peace-4-you) [inline bust-a-move-framework/src/test/java/bustamove/player/PlayerTest.java]: Dropped -> Score

File: bust-a-move-framework/src/main/java/bustamove/game/GameData.java (hunk 1 above)
Comment (mauricewillemsen) [inline bust-a-move-framework/src/main/java/bustamove/game/GameData.java]: Its part of the Observer now, so why not showing it on gamedata?

File: bust-a-move-framework/src/main/java/bustamove/game/GameData.java (hunk 2 above)
Comment (mauricewillemsen) [inline bust-a-move-framework/src/main/java/bustamove/game/GameData.java]: *

File: bust-a-move-framework/src/main/java/bustamove/game/GameData.java (hunk 1 above)
Comment (peace-4-you) [inline bust-a-move-framework/src/main/java/bustamove/game/GameData.java]: Ok

**Pull request merged**